warehouse/models/staging/payments/littlepay/_littlepay.yml

```diff
@@ -243,10 +235,20 @@ models:
         description: The number of records on which the `customer_id` shows up in the table
       - name: calitp_customer_id_rank
         description: A ranking of the records using the `customer_id`, ordered newest to oldest
+      - *lp_export_date
+      - *lp_export_ts
+      - *lp_line_number
+      - *payments_input_row_key
+      - name: _payments_key
```

Comment:
For this and any others with `unique_proportion`, can we use YAML anchors and just override minimally to change the unique proportion? Seems preferable to copying identical description many times, in case we want to modify. See for example: https://github.com/cal-itp/data-infra/blob/main/warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml#L366-L370